internal/commands/kubernetes/create.go

```diff
@@ -0,0 +1,249 @@
+package kubernetes
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/completion"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/output"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/ui"
+
+	"github.com/UpCloudLtd/upcloud-go-api/v4/upcloud"
+	"github.com/UpCloudLtd/upcloud-go-api/v4/upcloud/request"
+	"github.com/spf13/pflag"
+)
+
+// CreateCommand creates the "kubernetes create" command
+func CreateCommand() commands.Command {
+	return &createCommand{
+		BaseCommand: commands.New(
+			"create",
+			"Create a Kubernetes cluster",
+			`upctl kubernetes create \
+				--name my-cluster \
+				--network 03e5ca07-f36c-4957-a676-e001e40441eb \
+				--network-cidr "172.16.1.0/24" \
+				--node-group count=2,name=my-minimal-node-group,plan=K8S-2xCPU-4GB, \
+				--zone de-fra1`,
+			`upctl kubernetes create \
+				--name my-cluster \
+				--network 03e5ca07-f36c-4957-a676-e001e40441eb \
+				--network-cidr "172.16.1.0/24" \
+				--node-group \
+					count=4,kubelet-arg="log-flush-frequency=5s",label="owner=devteam",label="env=dev",name=my-node-group,plan=K8S-4xCPU-8GB,ssh-key="ssh-ed25519 AAAAo admin@user.com",ssh-key="/path/to/your/public/ssh/key.pub",storage=01000000-0000-4000-8000-000160010100,taint="env=dev:NoSchedule",taint="env=dev2:NoSchedule" \
```

Comment:
I wonder how this long example line will render in `--help` output and docs. Also something that we could polish closer to the beta release, though.